pkg/stub/handler.go

```diff
@@ -649,8 +741,6 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 		// Every time we see a change to the SamplesResource object, update the ClusterOperator status
 		// based on the current conditions of the SamplesResource.
 		err := h.cvowrapper.UpdateOperatorStatus(srcfg)
```

Comment:
if the samplesresource is `unmanaged` we should not be doing this, or at least i think we need to revisit what we want to report for status. Probably need to think about the implications for `removed` too.

Reply:
We can certainly discuss, but initially I think management state should not stop the reporting of operator status....the operator is available to receive input regardless of whether or not is processing changes ....

Reply:
I guess the real question is what should the ClusterOperator status be when the resource is unmanaged?

available:  reflect the last known state before the resource was marked unmanaged?  (is there any way this status can change once the resource is marked unmanaged?  I don't think so..?).  So this one probably "just works" w/o any special logic.

progressing:  do we keep monitoring resources we may have already installed (eg imagestreams we installed but weren't imported yet?) and continue to update this if/when those resources successfully import?  I think the answer may be yes...  Though I could also see an argument for immediately putting "progressing" to false when you mark the resource unmanaged, because the operator is no longer actively attempting to apply any changes you request.

failed:  presumably this never changes since we're not going to look at the config changes while unmanaged, so again it should never changed once the resource is marked unmanaged, so probably not a specific concern.

Reply:
available: agreement with what you said, but with one add-on:
- deleting the samplesresource with the current code "takes precedence" and results in "mangement" activity occurring .... in the vacuum of me iterating over this code that seemed valid, but admin it is nuanced enough that reasonable minds could disagree.

Your take?

failed: agreement with what you said

progressing: yeah, the either/or 's you mentioned don't necessarily have a clear winner in my mind at first blush as well;  at this point, I'd rather revisit the relationship between progressing and unmanaged *AFTER* we nail down what progressing will entail.

Thoughts?

Reply:
> deleting the samplesresource with the current code "takes precedence" and results in "mangement" activity occurring .... in the vacuum of me iterating over this code that seemed valid, but admin it is nuanced enough that reasonable minds could disagree.

hm.  yeah my inclination is to say if they have it marked "unmanaged" and then they delete it, we should not do anything (beyond deleting the samplesresource).  Unfortunately that means we're then going to create a new samplesresource which is managed, so the whole thing becomes a bit moot.

It may well be more consistent to treat deleting the resource as superceding unmanaged, as you suggest, since at least then the rest of the behavior is "logical".  So sure, let's go with that.

and i'm fine with deferring what to do w/ progressing until we actually have a valid progressing status.

Reply:
ok cool thx